src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -82,6 +81,23 @@ for further details.
   }
 }
 
+.sidebarContentWrapper {
+  position: relative;
+  padding: 24px;
+}
+
+.docsVersionSwitcherWrapper {
+  bottom: 0;
+  position: sticky;
+  width: 100%;
+  z-index: 2;
+
+  @media (--dev-dot-tablet-down) {
+    /* manually calculated based on .sidebarWrapper's padding-bottom */
+    bottom: -18px;
+  }
```

Comment:
📌 This doesn't seem super stable but I'm not sure what else to do for now.

Reply:
💭 🎨   I think this causes issues on tablet-sized viewports:

![CleanShot 2022-05-31 at 12 03 30@2x](https://user-images.githubusercontent.com/4624598/171218876-e250a5e7-49a4-4284-be43-f83a7522e31e.png)

Personally I'd remove this, and leave `bottom: 0` across viewports.

I think if there are positioning issues with that in place, they're like a result of 🎟️ [[sidebar slash top nav] fix mismatch in CSS height of top bars vs reality, especially on mobile](https://app.asana.com/0/1202114367927919/1202316169578550/f)? Maybe we should make that ticket more of a priority - in the video attached, I mention just forcing the height on the elements in question to prevent these issues. Maybe that'd be a decent temporary fix? Let me know what you think, I'd be happy to make that change 👍

Reply:
This sounds good to me! I'll update and remove it.

I think the steps forward are:

- Remove the alpha header above the main navigation bar
- Update all items in the navigation header to have static units in CSS (no `rem`s)
  - This will make sure the height always stays the same
  - I don't love this in the long term, but it's a good bandaid until we find a long term solution
- Store the height value of the navigation header in a variable as was previously done

Reply:
Addressed in: https://github.com/hashicorp/dev-portal/pull/491/commits/41fc487233f3908972f3646ffe68b3e133f1d882